src/utils/fees.ts

```diff
@@ -131,7 +131,7 @@ function getEstimatedWithdrawFee(
     fiatExchangeRates
   );
 
-  return exitFeeInFiat + withdrawFeeInFiat;
+  return exitFeeInFiat && withdrawFeeInFiat && exitFeeInFiat + withdrawFeeInFiat;
```

Comment:
I think this should now be abstracter to variables to make it easier to read

Reply:
I'm not seeing more variables, maybe for be more verbose we can add the if with the follow and we cover zero as well
```
if (exitFeeInFiat === undefined || withdrawFeeInFiat === undefined) {
    return undefined;
  }
  return exitFeeInFiat + withdrawFeeInFiat;
```

Reply:
As long as undefined is the only falsy value, sure. another option would be `!!exitFeeInFiat === false`

Reply:
I've pushed it with `undefined` that I think it's more exact